testing/unittests/jlibtests.cpp

```diff
@@ -3247,7 +3323,7 @@ class AtomicTimingStressTest : public CppUnit::TestFixture
     class Null
     {};
 
-    const unsigned numIterations = 1000000;
+    const unsigned numIterations = 1000;
```

Comment:
Should change back - or possibly reduce by a factor of 10 rather than 1000

Reply:
Done, and force-pushed